charts/rs-server-staging/values.yaml

```diff
@@ -37,6 +37,9 @@ app:
   endpointCatalog: http://rs-server-catalog-svc.processing.svc.cluster.local:8080  
   # -- Object Storage bucket for the catalog
   catalogBucket: rs-cluster-catalog
+  # -- Duration before the expiration of the access and refresh tokens
+  NbSecsBeforeAccessTokenExpire: 60
+  NbSecsBeforeRefreshTokenExpire: 60
```

Comment:
It's best to put 2 comments, one for each value, as the special prefix "# --" are used by the CI to automatically generated the docs :)